services/account-without-juggler/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "account",
+  "version": "1.0.0",
+  "description": "The Account microservice.",
+  "main": "index.ts",
+  "dependencies": {
+    "@loopback/testlab": "^4.0.0-alpha.5",
+    "@types/mocha": "^2.2.41",
```

Comment:
I think both `testlab` and `mocha` should go to `devDependencies`, shouldn't it? Why do you actually need `@types/mocha`?